abrt.spec.in

```diff
@@ -532,7 +533,7 @@ rm -f %{buildroot}%{_infodir}/dir
 make check|| {
     # find and print the logs of failed test
     # do not cat tests/testsuite.log because it contains a lot of bloat
-    find tests/testsuite.dir -name "testsuite.log" -print -exec cat '{}' \;
+    find src -name "test-suite.log" -print -exec cat '{}' \;
```

Comment:
But now any logs from the tests in tests/ will not be read, so the change should rather supplement what we already have (and remember that src/ is visited before tests/). Ideally, though, all tests would be in one place to avoid these headaches.